test/common.js

```diff
@@ -57,6 +57,20 @@ exports.validate = (t, report, options) => {
                 new RegExp('Node.js version: ' + process.version),
                 'NodeReport section contains expected Node.js version');
       }
+      if (options && options.commandline) {
+        if (this.isWindows()) {
+          // On Windows we need to strip double quotes from the command line in
+          // the report, and escape backslashes in the regex comparison string.
```

Comment:
n.b. I'm not seeing any double quotes in the reported command line in the reports generated from the tests on Windows.